docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+
+- As a multi-cluster operator, I would like to provide an EKS-like experience. Where the workload control plane nodes are joined to the management cluster and the control plane config isn’t exposed to the consumer of the workload cluster. This enables me as an operator to manage the control plane nodes for all clusters using tooling like prometheus and fluentd. I can also control the configuration of the workload control plane in accordance with business policy.
+
+### Multitenancy Management
+- As an operator of a management cluster, I want to control which users of management cluster can deploy new workload clusters, how many clusters they can deploy, and how many nodes/resources those clusters can use.
+
+- As an operator of a management cluster, I want to ensure that only the user who creates a new workload cluster (and some specific other users) can manage and access the workload cluster.
+
+- As an operator of a management cluster, I want the user who creates a new workload cluster to be able to give permission to other users to manage that cluster.
+
+- As an operator of a management cluster, I want to configure whether operators of workload clusters are allowed to open interactive shells onto those clusters machines.
+
+## Multi-cluster/Multi-provider
+
+### Managing Providers
+- As an operator, given I have a management cluster with at least one provider, I would like to install a new provider.
+
+- As an operator, given I have a management cluster with at least one provider, I would like to remove one of those providers and orphan any clusters provisioned by that provider.
+
+- As a multi-cluster operator, given that I have a single management cluster and that I have installed multiple providers and that one of those providers is malicious, I want that provider not to see IaaS secrets provided to any of the other providers.
+
+- As an operator, if I have a management cluster running a particular Cluster API version and a particular set of providers, then I want to plan an upgrade of Cluster API and the providers so that I upgrade one at a time and always end up with a compatible set of versions.
+
+### Creating Workload Clusters
+- As a multi-cluster operator, given that I have a management cluster, I want to create workload clusters across multiple providers with a consistent interface. For example, if I can create clusters on AWS without any manual intervention, I should have the same level of automation and lack of gotchas when using the VSphere provider.
+Keeping All the Clusters The Same
+
+- As a multi-cluster operator, given that I have a management cluster, I want to create workload clusters across multiple providers that are all similarly configured.
+
+- As a multi-cluster operator, given that I have deployed my clusters via Cluster API, I want to find general information (name, status, access details) about my clusters across multiple providers.
```

Comment:
Does this mean that you want to provide the ability of viewing all of the cluster resources in different cloud providers for the multi-cluster operator? Like viewing all of the cluster nodes, pods, applications etc? Or the general information just limited to cluster name, status and access detail?

Reply:
I don't think this would include what's running in the cluster, but rather it's going to be showing the status of those clusters from the information provided by Cluster API